test/e2e/provider_expectations.go

```diff
@@ -59,7 +60,7 @@ func (tc *testConfig) ExpectOneClusterObject() error {
 }
 
 func (tc *testConfig) ExpectAllMachinesLinkedToANode() error {
-	machineAnnotationKey := "machine"
+	machineAnnotationKey := "cluster.k8s.io/machine"
```

Comment:
Should we continue to test both until we get to a point where we know that the old key has gone away?

Reply:
The goal is to get away from `machine` asap.